app/code/Magento/Catalog/Controller/Adminhtml/Product/Save.php

```diff
@@ -140,7 +140,7 @@ public function execute()
 
                 $originalSku = $product->getSku();
                 $canSaveCustomOptions = $product->getCanSaveCustomOptions();
-                $product->save();
+                $this->productRepository->save($product);
```

Comment:
I believe the product repository does not cover all cases related to the admin. The only possible thing probably saving the product through product resource model.

Reply:
I think this statement should note/warnings in devdocs backend section too

Reply:
In my opinion we should use repositories for saving/retrieving models. But looks like repository do more than expected in this case

Reply:
Hi @jissereitsma,
As it doesn't work correctly - could you change to saving using the resource model?

Thank you!